### PR TITLE
Add section on how to customize the templates for all actors

### PR DESCRIPTION
**Pull request by mattwilkinsonn**
Related to [wash #373](https://github.com/wasmCloud/wash/issues/373).

Makes it easier for users to understand how to modify this template once it is generated.

Signed-off-by: Matt Wilkinson <matt@mattwilkinson.dev>

### CODE REVIEW

**Comment by mattwilkinsonn** [comment]
Just saw #65. Given that isn't ready to submit yet I'll leave this including the Makefile instructions, once this is submitted @brooksmtownsend can rebase and remove them

File: actor/echo/README.md [hunk 1]
```diff
@@ -1,15 +1,23 @@
...
+This actor echoes messages received in an `HttpRequest`.
+It is linked to the http server provider `wasmcloud:httpserver`.
 
 For each http request, the actor returns a json-formatted
 string containing fields from the request.
 
+## How do I customize this template to use other contracts & interfaces?
+
+- You can change what contracts this actor claims in `wasmcloud.toml` and the `Makefile`. In the future this will just be in `wasmcloud.toml`.
+- You will then need to change the dependencies in `Cargo.toml` to import the interfaces for the contracts you want. Optionally delete the `wasmcloud-interface-httpserver` dep if you're not using that contract.
```
Comment (stevelr) [inline actor/echo/README.md:12]: change "Optionally delete" to "Delete".
The list of contracts should match exactly what the actor needs to use.
(apply change to all files)

File: actor/echo/README.md [hunk 2]
```diff
@@ -1,15 +1,23 @@
...
+- Finally change the `src/lib.rs` file, changing/deleting the current interface import and `impl` block, while adding a new import & `impl` for any contracts you added!
```
Comment (stevelr) [inline actor/echo/README.md:13]: grammar nit: add comma(,) after Finally
(apply change to all files)

File: actor/echo/README.md [hunk 3]
```diff
@@ -1,15 +1,23 @@
...
+
 ### Using the included Github Actions
-If you store your source code on Github, we've gone ahead and included two actions: `build.yml` and `release.yml` under `.github/workflows`. The build action will automatically build, lint, and check formatting for your actor. The release action will automatically release a new version of your actor whenever code is pushed to `main`, or when you push a tag with the form `vX.Y.Z`. 
+
+If you store your source code on Github, we've gone ahead and included two actions: `build.yml` and `release.yml` under `.github/workflows`. The build action will automatically build, lint, and check formatting for your actor. The release action will automatically release a new version of your actor whenever code is pushed to `main`, or when you push a tag with the form `vX.Y.Z`.
```
Comment (stevelr) [inline actor/echo/README.md:17]: grammar nit: I think this reads more smoothly if we replace the line "If you store ...actions:" with

The generated project include two Github actions:

**Review by stevelr — CHANGES_REQUESTED** [review]

File: actor/echo/README.md (hunk 2 above)
Comment (mattwilkinsonn) [reply actor/echo/README.md:13]: Done.

File: actor/echo/README.md (hunk 3 above)
Comment (mattwilkinsonn) [reply actor/echo/README.md:17]: Done!

File: actor/echo/README.md (hunk 1 above)
Comment (mattwilkinsonn) [reply actor/echo/README.md:12]: Ohh I see what you mean - the "Optionally" and the "if you're not using that contract" imply that its optional to delete the dependency even in the case where you aren't using the contract.  Will fix.

**Review by brooksmtownsend — APPROVED** [review]
Great clarifications! Yeah, #65 will have to address this as well but that's fine and I'm happy to take that on

**Review by stevelr — APPROVED** [review]

**Comment by mattwilkinsonn** [comment]
@brooksmtownsend @stevelr one of you guys needs to merge, I don't have write access

**Pull request merged**